src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -41,8 +41,8 @@ public class CommandTestUtil {
     public static final String VALID_ADDRESS_CELINE = "Block 776, Cel Street 5";
     public static final String VALID_TAG_HUSBAND = "husband";
     public static final String VALID_TAG_FRIEND = "friend";
-    public static final String VALID_APPOINTMENT_FRIDAY = "23:59 FRI";
-    public static final String VALID_APPOINTMENT_SUNDAY = "08:00 SUN";
+    public static final String VALID_APPOINTMENT_FRIDAY = "18:00-20:00 FRI";
```

Comment:
Could be potentially worth it to make the start and end times more irregular times to produce better test cases perhaps? e.g. 1800 - 1837 etc.

Reply:
Making the start and end times more irregular does not improve the quality of the test cases so it is not worth it at this stage though it may be necessary later.

Reply:
No problem! Can put a pin in this for the future, in that case.